draft-ietf-quic-transport.md

```diff
@@ -2291,10 +2291,10 @@ Too Big messages.
 provisional until QUIC's loss detection algorithm determines that the packet is
```

Comment:
I'm operating here on the assumption that we are going to execute on the "application-layer shutdown" notion.  It's not at all clear to me that GOAWAY in its current form works as intended and this certainly messes up GOAWAY pretty badly.  My belief is that GOAWAY is much easier to implement at the HTTP layer where context is better understood.   You only need to concern yourself with requests and the pushes they trigger, which can simplify things a lot.  A fully generic graceful shutdown at the transport layer might not be achievable, despite the lofty goals of #604.